docs/changes.rst

```diff
@@ -22,6 +22,13 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.11.7 - 2017-06-27
+-------------------
+
+This release introduces new strategy - ``text_matching(regex)`` - that generates
```

Comment:
Use a Sphinx cross-reference:

    :func:`~hypothesis.strategies.text_matching`

(In fact, just read the new documentation guide in #707)